experimental/dds/tree2/src/test/shared-tree/editing.spec.ts

```diff
@@ -489,6 +489,288 @@ describe("Editing", () => {
 			expectJsonTree([tree, tree2], ["B", "A", "D", "C"]);
 		});
 
+		it("destination of move's ancestor deleted", () => {
+			const tree = makeTreeFromJson([{ foo: ["a"] }, {}]);
+			const tree2 = tree.fork();
+
+			const first: UpPath = {
+				parent: undefined,
+				parentIndex: 0,
+				parentField: rootFieldKeySymbol,
+			};
+
+			const second: UpPath = {
+				parent: undefined,
+				parentIndex: 1,
+				parentField: rootFieldKeySymbol,
+			};
+
+			const sequence = tree.editor.sequenceField({
+				parent: undefined,
+				field: rootFieldKeySymbol,
+			});
+
+			// Delete destination's ancestor concurrently
+			sequence.delete(1, 1);
+
+			tree2.editor.move(
+				{ parent: first, field: brand("foo") },
+				0,
+				1,
+				{ parent: second, field: brand("bar") },
+				0,
+			);
+
+			tree.merge(tree2);
+			tree2.rebaseOnto(tree);
+
+			expectJsonTree([tree, tree2], [{}]);
+		});
+
+		it("source of move's ancestor deleted", () => {
```

Comment:
```suggestion
		it("ancestor of move source deleted", () => {
```

Reply:
This one could also use a comment like "tests that a move is aborted if an ancestor of the moved nodes has been concurrently deleted."